src/api/apollo.js

```diff
@@ -14,7 +14,8 @@ export default function createApolloClient({
 	kvAuth0,
 	types,
 	uri,
-	fetch
+	fetch,
+	url = '',
```

Comment:
We might get more utility out of this if it was the current `route` object (or maybe even the entire `router`) instead of just the url string.

Reply:
I had assumed that the server wouldn't have access to a router object, but looking again, I see that we do use the router on the server entry as well. It looks like we can move the creation of the router around slightly and get the route object easily passed to the apollo `initState`. That will make it easier to grab the `setuiab` param, as well as other options in the future. I'll add this change to the ticket I'll be doing this week to extend the `setuiab` functionality. Thanks!